packages/frontend/src/renderer/containers/widgets/channels/CreateChannel.tsx

```diff
@@ -31,6 +47,25 @@ export const CreateChannel = () => {
   }, [channels])
 
   const createChannel = (name: string) => {
+    // Clear errors
+    if (error) {
+      dispatch(
+        errors.actions.clearError(error)
+      )
+    }
+    // Validate channel name
+    if (channels.some(channel => channel.name === name)) {
```

Comment:
I'm thinking if maybe we should validate it on waggle side as we are doing it with checking duplication of username.

Reply:
What are arguments behind this? Checking username duplication is different - it's owner who checks for it, and he is the one who always knows about every taken username, and registration module is in waggle. This case is different and checking it in waggle doesn't make things safer or better imo

Reply:
So what I was thinking when doing it this way was that the main purpose of the task is to present proper validation error message (and improve UX at this stage). We've been through a lot of discussions about uniqueness of channel's names/addresses and I think we haven't come with any ideal solution for now. 
What I did is relatively simple and it provides good user experience as it displays understandable error message when trying to add a channel with the same name as one of those which are already in our store, or subscribes to the channel we haven't yet subscribed to